sympy/holonomic/tests/test_holonomic.py

```diff
@@ -509,67 +514,103 @@ def test_to_hyper():
     q = besselj(0, x)
     assert p == q
 
-def test_to_sympy():
+def test_to_expr():
     x = symbols('x')
     R, Dx = DifferentialOperators(ZZ.old_poly_ring(x), 'Dx')
-    p = HolonomicFunction(Dx - 1, x, 0, 1).to_sympy()
+    p = HolonomicFunction(Dx - 1, x, 0, 1).to_expr()
     q = exp(x)
     assert p == q
-    p = HolonomicFunction(Dx**2 + 1, x, 0, [1, 0]).to_sympy()
+    p = HolonomicFunction(Dx**2 + 1, x, 0, [1, 0]).to_expr()
     q = cos(x)
     assert p == q
-    p = HolonomicFunction(Dx**2 - 1, x, 0, [1, 0]).to_sympy()
+    p = HolonomicFunction(Dx**2 - 1, x, 0, [1, 0]).to_expr()
     q = cosh(x)
     assert p == q
     p = HolonomicFunction(2 + (4*x - 1)*Dx + \
-        (x**2 - x)*Dx**2, x, 0, [1, 2]).to_sympy()
+        (x**2 - x)*Dx**2, x, 0, [1, 2]).to_expr()
     q = 1/(x**2 - 2*x + 1)
     assert p == q
-    p = from_sympy(sin(x)**2/x).integrate((x, 0, x)).to_sympy()
+    p = expr_to_holonomic(sin(x)**2/x).integrate((x, 0, x)).to_expr()
     q = (sin(x)**2/x).integrate((x, 0, x))
     assert p == q
     C_1, C_2, C_3 = symbols('C_1, C_2, C_3')
-    p = from_sympy(log(1+x**2)).to_sympy()
+    p = expr_to_holonomic(log(1+x**2)).to_expr()
     q = C_2*log(x**2 + 1)
     assert p == q
-    p = from_sympy(log(1+x**2)).diff().to_sympy()
+    p = expr_to_holonomic(log(1+x**2)).diff().to_expr()
     q = C_1*x/(x**2 + 1)
     assert p == q
-    p = from_sympy(erf(x) + x).to_sympy()
+    p = expr_to_holonomic(erf(x) + x).to_expr()
     q = 3*C_3*x - 3*sqrt(pi)*C_3*erf(x)/2 + x + 2*x/sqrt(pi)
     assert p == q
+    p = expr_to_holonomic(sqrt(x), x0=1).to_expr()
+    assert p == sqrt(x)
+    p = expr_to_holonomic(sqrt(1 + x**2)).to_expr()
+    assert p == sqrt(1+x**2)
+    p = expr_to_holonomic((2*x**2 + 1)**(2/3)).to_expr()
```

Comment:
I think you should use `S(2)/3` here, otherwise you get a floating point number. Floating point numbers can be handled differently, either now or in the future, so the test could fail, but `S(2)/3` should always work and be robust.